app/styles/headers/_fifth.scss

```diff
@@ -0,0 +1,51 @@
+.header__fifth {
+    height: 768px;
+    background: url(../img/header5/background.jpg) 100% no-repeat;
+    background-size: cover;
+    font-family: $font-face;
+    font-size: 17px;
+    color: white;
+}
+
+.fifth {
+    width: 66.1%;
```

Comment:
Can you please explain, why do you use this 66.1 magic number?  ✨

Reply:
I did it by Pixel Perfect and therefore I scale layout - long story.
I will correct this.